nexus_constructor/instrument_view_axes.py

```diff
@@ -0,0 +1,93 @@
+import struct
+from typing import List
+
+from PySide2 import QtCore
+from PySide2.Qt3DCore import Qt3DCore
+from PySide2.Qt3DExtras import Qt3DExtras
+from PySide2.Qt3DRender import Qt3DRender
+
+from nexus_constructor.axis_colors import AxisColors
+from nexus_constructor.line_geometry import LineGeometry
+from nexus_constructor.qentity_utils import (
+    set_material_properties,
+    add_qcomponents_to_entity,
+)
+
+
+class InstrumentViewAxes(object):
+    def __init__(self, component_root_entity: Qt3DCore.QEntity, line_length: float):
+        """
+        Class for housing the objects that create the axes in the instrument view.
+        :param component_root_entity: The root entity for the instrument view components.
+        :param line_length: The length of the line in the axes.
+        """
+
+        self.component_root_entity = component_root_entity
+        self.line_length = line_length
+
+        # The coordinate array given to the LineGeometry must be in the form
+        # [start_x, start_y, start_z, end_x, end_y, end_z]. All lines start at the origin so the first three elements
+        # in the lists for the three lines will be the same.
+        origin = [0, 0, 0]
+        x_line_vertices = origin + [line_length, 0, 0]
+        y_line_vertices = origin + [0, line_length, 0]
+        z_line_vertices = origin + [0, 0, line_length]
```

Comment:
This still seems to work fine without the origin part so I believe the comment is wrong